.github/workflows/kindIntegTest.yml

```diff
@@ -88,6 +88,8 @@ jobs:
         - additional_seeds
         - cluster_wide_install
         - config_change
+        - internode_encryption
```

Comment:
Remove, lets not re-enable it ;)

Reply:
Just my way of saying, are you sure.... 😆